solutions/search/esql-search-tutorial.md

```diff
@@ -5,23 +5,21 @@ applies_to:
 navigation_title: Search and filter with ES|QL
 ---
 
-# Tutorial: Search and filter with {{esql}}
+# Search and filter with {{esql}}
```

Comment:
Consider keeping "Tutorial" in the title to identify the purpose of the page more easily.

Reply:
Sounds good.